users/models/identity.py

```diff
@@ -812,6 +813,16 @@ def to_opengraph_dict(self) -> dict:
 
     ### Mastodon Client API ###
 
+    @cache_memoize(120)
+    def mention_to_mastodon_json(self):
```

Comment:
This should be called `to_mastodon_mention_json`

Reply:
Fixed